app/src/main/res/values-es-rES/strings.xml

```diff
@@ -1764,4 +1764,20 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <string name="deprecated_version_dialog_store_button">Instalar…</string>
     <string name="security_options_permission_fine_location_warning">Wolvic sólo puede obtener tu ubicación aproximada. Esto se puede cambiar en la configuración del sistema.</string>
     <string name="hamburger_menu_toggle_passthrough">Passthrough</string>
+    <string name="user_feedback_tooltip">Compartir ideas y opiniones</string>
+    <string name="rCN_first_launch_clickable_privacy_text">Política de privacidad</string>
+    <string name="rCN_first_launch_agree">Aceptar</string>
+    <string name="rCN_first_launch_disagree">No estoy de acuerdo</string>
+    <string name="rCN_first_launch_confirmation_body">\\Si seleccionas \"No estoy de acuerdo\", lamentablemente ya no podremos brindarle los servicios.
+\n
+\nPuede leer el Acuerdo de usuario y la Política de privacidad completos.</string>
+    <string name="rCN_first_launch_confirmation_agree">Aceptar y continuar</string>
+    <string name="rCN_first_launch_title">Acuerdo del usuario y la notificación de la política de privacidad</string>
```

Comment:
"Acuerdo del usuario" ? Wouldn't be better "Acuerdo de usuario" ? Just asking.

Reply:
Good afternoon:

You are right I inadvertently increased by one l.

I have already modified it.

Sorry for the mistake.

Best regards